src/hooks/useSingleProduct.ts

```diff
@@ -12,6 +12,11 @@ const fetchProductData = async (sku: string): Promise<Product> => {
     headers: { Authorization: `Bearer ${jwtToken}` }
   });
 
+  if (!response.ok) {
+    console.log('error');
+    throw new Error(`Status Code ${response.status}.  Error fetching sku: ${response.statusText}.`);
```

Comment:
Instead of a string here and doing string matching in the error handler, we could create the following type to allow us to easily access this info (I'm writing this on the fly without testing it so it may need to be tweaked - maybe this should be a class?)

```
interface HttpError extends Error {
    status: number
    statusText: string
}
```